packages/safe-apps-test-app/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@gnosis.pm/safe-apps-test-app",
+  "version": "0.0.1",
+  "description": "Safe App for testing purposes",
+  "author": "Safe Foundation",
+  "license": "MIT",
+  "private": true,
+  "homepage": ".",
+  "dependencies": {
+    "@gnosis.pm/safe-apps-sdk": "^7.8.0",
```

Comment:
We can link this to the local repo: 

"@gnosis.pm/safe-apps-sdk": "file:../safe-apps-sdk"

Ideally after each `build:sdk` we should copy the `dist` directory from `safe-apps-sdk` to the `safe-apps-test-app`  corresponding directory inside node_modules. This way we could make the testing faster. But anyway this is a nice to have we can implement in any moment ...

Reply:
https://classic.yarnpkg.com/lang/en/docs/workspaces/#toc-why-would-you-want-to-do-this

Yarn workspaces already do it